templates/CommentsInterface_singlecomment.ss

```diff
@@ -11,32 +11,32 @@
 		<% end_if %>
 	</p>
 <% end_if %>
-
-<div class="comment-text" id="<% if $isPreview %>comment-preview<% else %>$Permalink<% end_if %>">
+	
+<div class="comment-text well" id="<% if $isPreview %>comment-preview<% else %>$Permalink<% end_if %>">
```

Comment:
We can drop the `well` from here. It doesn't seem relevant to this template is it?